modelator/const_values.py

```diff
@@ -1,8 +1,19 @@
 # model checkers
+import os
+import appdirs
+
+
 APALACHE = "apalache"
 TLC = "tlc"
 
-DEFAULT_APALACHE_JAR = "jars/apalache.jar"
+APALACHE_RELEASE_URL = (
+    "https://github.com/informalsystems/apalache/releases/download/v0.25.0/apalache.zip"
+)
+DEFAULT_APALACHE_VERSION = "0.25.0"
+DEFAULT_APALACHE_LOCATION = os.path.join(appdirs.user_data_dir(__package__), "checkers")
+DEFAULT_APALACHE_JAR = os.path.join(
+    DEFAULT_APALACHE_LOCATION, "apalache", "lib", "apalache.jar"
+)
```

Comment:
Does it make sense to maintain a different version of Apalache here - so that a user can configure a specific Apalache version to use?

Reply:
I would not give that option, at least for now: I would only allow it once we have a clear idea what versions we are supporting (e.g., when experimenting, I figured out something breaks for us with the newest version of Apalache)